project/source/engine/KeyStates.h

```diff
@@ -23,18 +23,18 @@ enum ShortcutKey
 typedef delegate<void(int)> KeyDownCallback;
 
 //-----------------------------------------------------------------------------
-// stan klawiatury
+// keyboard state
 struct KeyStates
 {
 	KeyStates();
 
-	// proste sprawdzanie czy klawisz zosta� wci�ni�ty, wyci�ni�ty, jest wci�ni�ty, jest wyci�ni�ty
+	// key pressed/unpressed state check. If it is pressed - it is not pressed.
```

Comment:
To drugie zdanie nie ma sensu 😜A poza tym po co wymyślasz "unpressed" jak niżej masz jasno napisane że to "released".

Simple key checking: pressed, released, up or down

Reply:
myślisz że tak łatwo Cię zrozumieć